src/components/dashboard/main-content/course-enrollments/course-cards/CompletedCourseCard.jsx

```diff
@@ -1,36 +1,143 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
 import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
+import { sendTrackEvent } from '@edx/frontend-platform/analytics';
+import { courseEnded } from '../../../../../utils/common';
+import ContinueLearningButton from './ContinueLearningButton';
+
+import {
```

Comment:
tiny nit: swap these action imports with the `UnarchiveModal` import so that components are all grouped together, and then misc things like actions at the end of the imports.